djpsa/halo/records/ticket/sync.py

```diff
@@ -170,3 +174,25 @@ def _assign_field_data(self, instance, json_data):
             instance.agent = None
         if instance.client_id == UNASSIGNED_CLIENT_ID:
             instance.client = None
+
+    def get_related_synchronizers(self, instance):
+        """
+        Return a list of related synchronizers.
+        """
+        sync_classes = []
+
+        appointment_sync = AppointmentSynchronizer(full=True,
```

Comment:
Do you like this formatting? The arguments are indented so far I think it makes it hard to read.

Reply:
Well if it's intentional I don't insist on changing it.

Reply:
Oh wait, no I see what you mean. No, that was an accident.